src/libraries/System.IO.FileSystem/tests/FileStream/CharacterDevice.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.IO.Tests
```

Comment:
Can this be part of the `ctor_*` tests that gets run from different APIs?

Reply:
It was hard to figure out exacty how much is covered by those clases and I didn't want to end up with a ton of duplication (caused by the inheritance). 
I would prefer to wait for @adamsitnik to come back to refactor the tests since he is more familiar with them.